packages/react-ui/src/index.d.ts

```diff
@@ -35,6 +40,10 @@ export {
   TableWidgetUI,
   LegendWidgetUI,
   LEGEND_TYPES,
+  LegendCategories,
+  LegendIcon,
+  LegendProportion,
+  LegendRamp,
+  LegendWrapper,
```

Comment:
Until we decide about 'public' api, we should export 'undocument' / internal methods with underscore, as eg `_LegendCategories`